config/imager.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * Configuration file for Imager
+ *
+ * Multi-environment settings work in this file the same way as in general.php or db.php
+ */
+
+return [
+    // Global settings
+    '*' => [
+        'imagerSystemPath' => '@assetBasePath/imager/',
+        'imagerUrl' => '@assetBaseUrl/imager/',
+
+        'jpegQuality' => 85,
+        'allowUpscale' => false,
+        'interlace' => 'plane',
+        'removeMetadata' => true,
+        'cacheDuration' => 60*60*24*365*10, // 10 years
+
+        'useCwebp' => true,
+        'cwebpPath' => '/usr/bin/cwebp',
+
+        'optimizerConfig' => [
+            'jpegoptim' => [
+                'optionString' => '-s -m85 -T1',
+                'extensions' => ['jpg'],
+                'path' => '/usr/bin/jpegoptim',
+            ],
+            'optipng' => [
+                'extensions' => ['png'],
+                'path' => '/usr/bin/optipng',
+                'optionString' => '-o2',
+            ],
+        ],
+
+        'clearKey' => 'SH0ZDR%PVbqwg%k7lMt&t0tyHWRp0Bh@',
```

Comment:
This value is supposed to be unique and secret for each project. Should probably disable (comment out) for the boilerplate.

Reply:
Oh, okay. It was originally the same for the last project we did and the second last project, so I thought you considered it unnecessary to change it.

Reply:
Commented out that line in the next commit.